components/landing/QuickStart/StartLocally.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <section class="start-locally">
+    <h3 class="start-locally__title">
+      Start locally
+    </h3>
+    <p class="start-locally__description">
+      Although it isn’t required we recommend using
+      <AppLink
+        class="start-locally__link"
+        :url="downloadAnaconda.url"
+      >
+        {{ downloadAnaconda.label }}
+      </AppLink>
+      to cleanly separate Qiskit from other applications and improve your experience.
+    </p>
+    <div class="start-locally__options">
+      <div
+        v-for="optionsBlock in installOptions"
+        :key="optionsBlock.alias"
+      >
+        <h4 class="start-locally__option-title">
+          {{ optionsBlock.title }}
+        </h4>
+        <cv-button-set :class="`start-locally__options-group start-locally__options-group_${optionsBlock.alias}`">
+          <cv-button
+            v-for="option in optionsBlock.elements"
+            :key="option"
+            :title="option"
+            class="start-locally__option"
+            :class="{ 'start-locally__option_active': isActive (optionsBlock.alias, option) }"
+            @click="selectOption(optionsBlock.alias, option)"
+          >
+            {{ option }}
+          </cv-button>
+        </cv-button-set>
+      </div>
+      <div>
+        <h4 class="start-locally__option-title">Terminal</h4>
+        <SyntaxHighlight
+          :label="title"
+          :code="getCodeToInstallQiskit()"
+          lang="console"
+          btn-on-bottom
+          white-design
+        />
+      </div>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import SyntaxHighlight from '~/components/ui/SyntaxHighlight.vue'
+import AppLink from '~/components/ui/AppLink.vue'
+import { DOWNLOAD_ANACONDA_LINK } from '~/constants/appLinks'
+
+@Component({
+  components: { SyntaxHighlight, AppLink },
+  head () {
+    return {
+      link: [
+        { rel: 'stylesheet', href: 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/styles/atom-one-dark.min.css' }
+      ]
+    }
+  }
+})
+export default class extends Vue {
+  downloadAnaconda = DOWNLOAD_ANACONDA_LINK
+  installOptions = [
+    {
+      title: 'Languages',
+      alias: 'languages',
+      elements: ['Python 3.5+']
+    },
+    {
+      title: 'Qiskit Install',
+      alias: 'qiskit-install',
+      elements: ['Stable (Recommended)', 'Source']
+    },
+    {
+      title: 'Operating System',
+      alias: 'os',
+      elements: ['Linux', 'Mac', 'Windows']
+    }
+  ]
+
+  activeOptions = {
```

Comment:
Instead of active, can you use `selected`. That way it follows the meaning when you do `selectOption`.